package.json

```diff
@@ -8,9 +8,9 @@
     "url": "https://github.com/kuzzleio/kuzzle-plugin-device-manager"
   },
   "scripts": {
+    "docker:up": "docker-compose up",
```

Comment:
This script has the same name in every other repository. To preserve consistency it should stay like this or be changed in every other repository (I'm not even sure it's a good idea since our users are used to this scrip)

You can rather create an alias if you want